provider/resource.go

```diff
@@ -32,6 +32,7 @@ import (
 	"github.com/hashicorp/terraform-provider-azurerm/shim"
 	"github.com/pulumi/pulumi-azure/provider/v5/pkg/version"
 	"github.com/pulumi/pulumi-terraform-bridge/v3/pkg/tfbridge"
+	"github.com/pulumi/pulumi-terraform-bridge/v3/pkg/tfbridge/x"
```

Comment:
x?

Reply:
The API is considered "experimental", and `/x` is the standard go way to indicate that. We have this deployed in most of our providers, so this is more about out-of-company api flexibility at this point.